src/schema/imported/manifest.json

```diff
@@ -31,6 +31,7 @@
         },
         "name": {
           "type": "string",
+          "description": "Name must be at least 2, at should be at most 75 characters",
```

Comment:
FWIW I fixed this typo in https://bugzilla.mozilla.org/show_bug.cgi?id=1950334

Reply:
ty, I'll leave that fixed typo to be covered from the next import for 137 data once it gets to beta in the next couple of weeks.